pkg/updatestatus/nodeinformer.go

```diff
@@ -107,89 +117,23 @@ func (c *nodeInformerController) sync(ctx context.Context, syncCtx factory.SyncC
 
 	var msg informerMsg
 	switch t {
+	case eventKindName:
```

Comment:
So now the controller reacts to basically two kinds of triggers:

1. Syncs triggered for standard kube informer "events" on watched resources (nodes, mcp, mc)
2. Synthetic triggers (right now we have just one)

I think this should be documented in `sync` godoc, and ideally we should have a list of synthetic triggers and what they do (very briefly), even when we only have just one now.

Ideally the two cases should be recognizable in the code, too - it woudl be great if the `...KindName` was used only for the "we are reconciling a change on a watched resource" triggers, and we'd use something else (`syntheticKeyX`?) for the synthetic ones.

Reply:
Done.
Use `syncSyntheticKey` as the function name.